### ISSUE TITLE
Variant display skewed if variants have the same width

### ISSUE THREAD

**Issue by nfourtythree** [nystudio107/craft3-imageoptimize#35]
Hey @khalwat 

Great plugin as ever! Just noticed that when previewing the variants on an asset, if all the variants have the same width it seems to only be using the last transform on all the previews.

This causes a bit of an issue with the previews looking skewed if the aspect ratios are different.

Have put together a short video that should show the issue: http://x.n43.me/3Y2O020C0M1T

Keep up the great work!

**Comment by khalwat** [comment]
Seems a little odd to have variants that have the same width? The idea is that we're making responsive image variants of different widths corresponding to the sizes we want at various @media breakpoints?

**Comment by khalwat** [comment]
It actually indexes the responsive image variants based on the width, so I think it's probably not going to work... or be very useful for this use-case (the `srcset` won't work either).

**Comment by nfourtythree** [comment]
Yea I definitely knew that it was using it in a way that was maybe not fully intended. As the plan is that we are using configs that are pulled out in the templates to determine the transform.

So the idea that I displayed above was just for a CMS user to have a quick visual clue as to what the image might look like with the set focal point in different image aspect ratios.

Again I understand that I am not using it in its fully intended purpose, just thought I would highlight a little thing I spotted.

Still, have to say that it is an absolutely fantastic plugin and is a big help in projects!
